src/scales/scale.time.js

```diff
@@ -480,11 +480,16 @@ module.exports = function(Chart) {
 			var timestamps = [];
 			var datasets = [];
 			var labels = [];
+			var rawLabels = [];
 			var i, j, ilen, jlen, data, timestamp;
 
 			// Convert labels to timestamps
 			for (i = 0, ilen = chart.data.labels.length; i < ilen; ++i) {
-				labels.push(parse(chart.data.labels[i], me));
+				timestamp = parse(chart.data.labels[i], me);
+				if (timestamp !== null) {
```

Comment:
@etimberg do we support `null` in the `data.labels` array (what use case)? If not, I would consider that as an error and would avoid duplicating the `labels` timestamps array with `rawLabels`.